.github/workflows/test.yml

```diff
@@ -162,3 +162,28 @@ jobs:
           JF_URL: ${{ secrets.PLATFORM_URL }}
```

Comment:
In the `GitHub Integration Tests`, let's remove the FROGBOT_TESTS_GITHUB_TOKEN from the job's environment variables.